.github/workflows/tests.yml

```diff
@@ -187,11 +187,17 @@ jobs:
       matrix:
         include:
           - runner: "12-arm64"
+            ephemeral: false
           - runner: "12-${{github.run_id}}-${{github.run_attempt}}"
+            ephemeral: true
           - runner: "11-arm64"
+            ephemeral: false
           - runner: "11-${{github.run_id}}-${{github.run_attempt}}"
+            ephemeral: true
           - runner: "10.15-${{github.run_id}}-${{github.run_attempt}}"
+            ephemeral: true
           - runner: ${{needs.setup_tests.outputs.linux-runner}}
+            ephemeral: true
```

Comment:
Might still need cleanup for linux-self-hosted-1, as $HOME is mounted I think.

Reply:
`--local` writes files to the current working directory, I think.

Reply:
Yeh, it does, rather than the repo. This was just done to make `git clean` not nuke the in-progress stuff.

Reply:
I ask because I was very careful about the use of linux-self-hosted-1 in the publish workflow because the home directory is not cleared between runs: https://github.com/Homebrew/homebrew-core/blob/1200188b20925689100427db2f27ae7e3567ec3c/.github/workflows/publish-commit-bottles.yml#L17

Reply:
@Bo98 is it truly ephemeral if not cleared between runs?

Reply:
linux-self-hosted-1 has never been advertised as truly ephemeral. It's recreated on a 24-hour schedule.

I would like to make it fully ephemeral though. I could integrate that into the CI orchestrator.

Reply:
Yeh, that'd be good. I think it still needs the cleanup steps here until it's fully ephemeral.